build/build.go

```diff
@@ -82,6 +86,29 @@ func taskFmt() goyek.Task {
 	}
 }
 
+func taskMarkdownLint() goyek.Task {
+	return goyek.Task{
+		Name:  "markdownlint",
+		Usage: "markdownlint-cli",
```

Comment:
This should note that `docker` is required.

I'm not sure whether also to provide a flag that allows to disable the docker call (enabled by default). I have thoughts for and against that.

Reply:
If you have docker as well then I would rather not add anything.

Some time ago I was thinking of adding a global parameter that would ignore calling some provided tasks...

Reply:
Check: https://github.com/goyek/goyek/pull/96/commits/d52f119df840bae0ba4aca1b0031e52b747b05e1